AGPs/AGP-CS.md

```diff
@@ -0,0 +1,64 @@
+---
+AGP: N/A
+Title: Develop and launch Aragon Chain
+Author: Aidan Hyman & Gregory Markou
+Status: Stage III
+Track: Finance
+Created: 2019-10-03
+---
+
+# Develop and launch Aragon Chain
+
+## Address of the transfer recipient
+
+TBD
```

Comment:
I recommend that this section be completed with a recipient address before the Stage V deadline.